lib/pages/other/WebView.dart

```diff
@@ -7,37 +9,70 @@ import 'package:app/assets/components/customAppBar.dart';
 import 'package:stacked/stacked.dart';
 import 'package:app/viewmodels/web_view_model.dart';
 
-class WebViewArgumnets {
+class WebViewArguments {
   final String initialUrl;
 
-  WebViewArgumnets(this.initialUrl);
+  WebViewArguments(this.initialUrl);
 }
 
-class WebViewPage extends StatelessWidget {
-  final WebViewArgumnets args;
+class WebViewPage extends StatefulWidget {
+  final WebViewArguments args;
   WebViewPage(this.args);
+  
+
+  @override
+  _WebViewPageState createState() => _WebViewPageState();
+}
+
+class _WebViewPageState extends State<WebViewPage> {
+
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: CustomAppBar(
+      appBar: customAppBar(
         backButtonText: "Back",
         context: context,
-      ),
-      body: SafeArea(
-        child: ViewModelBuilder<WebViewViewModel>.reactive(
-            viewModelBuilder: () => WebViewViewModel(),
-            builder: (context, model, child) {
-              return WebView(
-                javascriptMode: JavascriptMode.unrestricted,
-                initialUrl: args.initialUrl,
-                onWebResourceError: (error) {
-                  model.onWebError(args.initialUrl);
-                },
+        actions: [
+         Padding(
+           padding: const EdgeInsets.all(12.0),
+           child: DropdownButton<String>(
+             iconEnabledColor: Theme.of(context).primaryColor,
+             isDense: true,
+             items: <String>['Open'].map((String stringValue) {
+                 return DropdownMenuItem<String>(
```

Comment:
I like the idea of the dropdown menu, but is there a way we can change the icon? Currently, the icon seems to be the arrow but also a line underneath (I can send a screenshot it that helps). I think the best would be the 3 vertical dots icon.

Reply:
Ok,That's great.I'll update it.